src/components/MainNav/AppList/AppList.js

```diff
@@ -186,20 +190,20 @@ class AppList extends Component {
     return (
       <nav className={css.appList}>
         <ul className={css.navItemsList}>
-          { renderNavButtons() }
+          {renderNavButtons()}
         </ul>
         <div className={css.navListDropdownWrap}>
           <Dropdown tether={tether} dropdownClass={css.navListDropdown} open={this.state.open} id={dropdownId} onToggle={toggleDropdown}>
-            { renderDropdownToggleButton() }
+            {renderDropdownToggleButton()}
             <DropdownMenu data-role="menu" onToggle={toggleDropdown}>
-              { this.focusTrap(this.focusDropdownToggleButton) }
+              {this.focusTrap(this.focusDropdownToggleButton)}
               <AppListDropdown
                 listRef={this.dropdownListRef}
                 apps={apps}
                 dropdownToggleId={dropdownToggleId}
                 toggleDropdown={toggleDropdown}
               />
-              { this.focusTrap(this.focusFirstItemInList) }
+              {this.focusTrap(this.focusFirstItemInList)}
```

Comment:
Isn't it better to destructure this stuff? Especially considering that this is already done with other properties of 'this' object at the top of render method.